src/components/Loader/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const AUTO_LOADER_SIZE = 'auto'
+export const SM_LOADER_SIZE = 'sm'
+export const MD_LOADER_SIZE = 'md'
+export const LG_LOADER_SIZE = 'lg'
```

Comment:
```suggestion
export const LOADER_SIZE_LG = 'lg'
```
Предлагаю использовать формат как в panel-merchant: СУЩНОСТЬ_ПАРАМЕТР_ЗНАЧЕНИЕ